backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/appsearchmodal/internal/BpkSection.kt

```diff
@@ -70,17 +72,32 @@ internal fun BpkSectionItem(item: BpkItem, modifier: Modifier = Modifier) {
             .fillMaxWidth()
             .clickable(onClick = item.onItemSelected),
         verticalAlignment = Alignment.CenterVertically,
-        horizontalArrangement = Arrangement.spacedBy(BpkSpacing.Base),
+        horizontalArrangement = Arrangement.SpaceBetween,
     ) {
-        BpkIcon(
-            icon = item.icon,
-            contentDescription = null,
-            size = BpkIconSize.Large,
-
-        )
-        Column {
-            BpkText(text = item.title)
-            BpkText(text = item.subtitle)
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.spacedBy(BpkSpacing.Base),
+        ) {
+            BpkIcon(
+                icon = item.icon,
+                contentDescription = null,
+                size = BpkIconSize.Large,
+            )
+            Column {
+                BpkText(text = item.title)
+                BpkText(
+                    style = BpkTheme.typography.footnote,
+                    text = item.subtitle,
+                )
+            }
+        }
+        Box(
+            modifier = Modifier.padding(BpkSpacing.Base),
```

Comment:
this will give it double the padding on the end (which you can also see in the screenshot) - if you follow what i suggest above `spacedby` will take care of the spacing and this box is no longer needed

Reply:
this also added extra vertical spacing as seen by the snapshot test changes

Reply:
Updated above mention.